docs/user-guide/gateway.md

```diff
@@ -0,0 +1,221 @@
+    direction TB
+    Server_01["Server-1"]
+    Server_02["Server-2"]
+    Server_03["Server-3"]
+    Server_04["Server-4"]
+    Server_05["Server-5"]
+    Server_06["Server-6"]
+end
+
+%% Connections
+
+%% Gateway_01 -> Spines
+Gateway_01 --- Spine_01
+Gateway_01 --- Spine_02
+
+Spine_01 --- Leaf_01
+Spine_01 --- Leaf_02
+Spine_01 --- Leaf_03
+Spine_02 --- Leaf_01
+Spine_02 --- Leaf_02
+Spine_02 --- Leaf_03
+
+%% Leaves -> Servers
+Leaf_01 --- Server_01
+Leaf_01 --- Server_02
+Leaf_02 --- Server_03
+Leaf_02 --- Server_04
+Leaf_03 --- Server_05
+Leaf_03 --- Server_06
+
+subgraph Legend["Network Connection Types"]
+    direction LR
+    %% Create invisible nodes for the start and end of each line
+    L1(( )) --- |"Gateway Links"| L2(( ))
+    L7(( )) --- |"Fabric Links"| L8(( ))
+    L9(( )) --- |"Server Links"| L10(( ))
+end
+
+class Gateway_01 gateway
+class Spine_01,Spine_02 spine
+class Leaf_01,Leaf_02,Leaf_03 leaf
+class Server_06,Server_05,Server_04,Server_03,Server_02,Server_01 server
+class L1,L2,L7,L8,L9,L10 hidden
+class Legend legendBox
+linkStyle default stroke:#666,stroke-width:2px
+linkStyle 0,1 stroke:#CC9900,stroke-width:2px
+linkStyle 2,3,4,5,6,7 stroke:#CC3333,stroke-width:2px
+linkStyle 8,9,10,11,12,13 stroke:#6C8EBF,stroke-width:2px
+linkStyle 14 stroke:#CC9900,stroke-width:2px
+linkStyle 15 stroke:#CC3333,stroke-width:2px
+linkStyle 16 stroke:#6C8EBF,stroke-width:2px
+
+%% Make subgraph containers invisible
+style Spines fill:none,stroke:none
+style Leaves fill:none,stroke:none
+style Servers fill:none,stroke:none
+
+
+```
+
+## Gateway Peering
+
+Just as [VPC Peerings](vpcs.md#vpcpeering) provide VPC to VPC connectivity by way of the switches in the fabric, gateway peerings provide connectivity via the gateway nodes.
+Gateway services can be inserted between a pair of VPCs or a VPC and an external using a Gateway Peering.
+Each peering can be configured to provide the necessary services for traffic that uses that peering.
+
+### Simple Gateway Peering Between VPCs
+
+```{.yaml .annotate linenums="1" title="gw-peer.yaml"}
+apiVersion: gateway.githedgehog.com/v1alpha1
+kind: Peering
+metadata:
+  name: vpc-1--vpc-2
+  namespace: default
+spec:
+  peering:
+    vpc-1:
+      expose:
+        - ips:
+          - cidr: 10.0.0.0/24 # Expose all IP address in the 10.0.0.0/24 CIDR block to vpc-2
+    vpc-2:
+      expose:
+        - ips:
+          - cidr: 192.168.0.0/16 # Expose all IP addresses in the 192.168.0.0/16 CIDR block to vpc-1
+```
+
+A simple peering with no services deployed between the VPCs. This traffic will
+transit the gateway node(s).
+
+### Gateway Peering with stateless NAT
+
+```{.yaml .annotate linenums="1" title="sl-gw-peer.yaml"}
+apiVersion: gateway.githedgehog.com/v1alpha1
+kind: Peering
+metadata:
+  name: vpc-1--sl-nat--vpc-2
+  namespace: default
+spec:
+  peering:
+    vpc-1:
+      expose:
+        - ips:
+          - cidr: 10.0.1.0/24 # Expose all IP address in the 10.0.1.0/24 CIDR block to vpc-2
```

Comment:
```suggestion
          - cidr: 10.0.1.0/24 # Expose all IP addresses in the 10.0.1.0/24 CIDR block to vpc-2
```

They're not directly exposed, given that we use NAT: vpc-2 won't ever see these. Could we phrase this differently?